modules/basis_universal/SCsub

```diff
@@ -48,6 +48,27 @@ if env.dev_build:
 
 env_thirdparty = env_basisu.Clone()
 env_thirdparty.disable_warnings()
+
+# Disable unneeded features to reduce binary size.
+# <https://github.com/BinomialLLC/basis_universal/wiki/How-to-Use-and-Configure-the-Transcoder>
+env_thirdparty.Append(
+    CPPDEFINES=[
+        # Storage formats.
+        # Godot only implements `.basis` support through basis_universal.
+        # Support for `.ktx` files are implemented with a direct libktx implementation.
+        # Building the encoder requires `BASISD_SUPPORT_KTX2` to be enabled,
+        # so we can only disable Zstandard compression for `.ktx` files
+        # (this is not used in `.basis` files).
+        ("BASISD_SUPPORT_KTX2_ZSTD", 0),
+        # GPU compression formats.
+        ("BASISD_SUPPORT_ATC", 0),  # Proprietary Adreno format not supported by Godot.
+        ("BASISD_SUPPORT_ETC2_EAC_RG11", 0),  # Godot's implementation of Basis Universal doesn't request RG ETC2.
```

Comment:
```suggestion
```

After #90170 Godot requires ETC2 R/RG.

I've tested this PR on Windows 10 and everything works correctly. These formats are not requested by Godot at any point, so I'm assuming other platforms are unaffected by this change as well.